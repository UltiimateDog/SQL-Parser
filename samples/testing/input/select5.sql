SELECT * FROM Enrolled WHERE Enrolled.A = 1 AND Enrolled.H < 90 AND Enrolled.E = 101;