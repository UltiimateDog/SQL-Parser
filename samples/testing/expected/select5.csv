1, 101, 75
